apptainer/demos/openradioss/README.md

```diff
@@ -0,0 +1,370 @@
+
+```bash
+vortex-radioss
+Apptainer> from vortex_radioss.animtod3plot.Anim_to_D3plot import readAndConvert
+Apptainer> readAndConvert("2010-toyota-yaris-detailed-v2j/YarisOpenRadioss")
+```
+
+Create archirve with d3plot files
+
+```bash
+tar zcvf data.tgz 2010-toyota-yaris-detailed-v2j/YarisOpenRadioss.d3plot*
+```
+
+Download data from login node to local computer to visualize data with LS-PrePost.
+
+```bash
+gcloud compute scp $(gcloud compute instances list --filter="NAME ~ login" --format="value(NAME)"):/home/_USERNAME_/data.tgz . --tunnel-through-iap
+```
+
+## Visualization (ParaView)
+
+### Server
+
+Now create a `vis.sh` file that specifies the compute environment for the visualization and to start ParaView server.
+
+```bash
+cat << EOF > vis.sh
+#!/bin/bash
+#SBATCH --partition=gpu
+#SBATCH --nodes=1
+#SBATCH --gpus-per-node=1
+
+cd ~/2010-toyota-yaris-detailed-v2j
+apptainer run --nv ~/bin/paraview pvserver
+EOF
+```
+
+Set up access to the Artifact Registry repository
+
+```bash
+export REPOSITORY_URL=#ARTIFACT REGISTRY REPOSITORY URL# e.g. oras://us-docker.pkg.dev/myproject/sifs
+```
+
+```bash
+apptainer registry login \
+--username=oauth2accesstoken \
+--password=$(gcloud auth print-access-token) \ 
+${REPOSITORY_URL}
+```
+
+Pull the `paraview` container and put it in `~/bin`
+```bash
+apptainer pull ${REPOSITORY_URL}/paraview:latest
+mkdir ~/bin
+mv paraview_latest.sif ~/bin/paraview
+```
+
+Now you are ready to actually run the ParaView server with the command
+```bash
+sbatch vis.sh
+```
+
+You should see output similar to
+```
+Waiting for client...
+Connection URL: cs://openradios-gpunodeset-0:11111
+Accepting connection(s): openradios-gpunodeset-0:11111
+```
+
+### Client
+
+On a workstation, create an `ssh tunnel` using the command
+```
+gcloud compute start-iap-tunnel openradios-gpunodeset-0 11111 \
+    --local-host-port=localhost:11111
+```
+
+Now start the `paraview` client that you downloaded. The first time you start it the client may take a few minutes to come up. Eventually you should
+see the ParaView UI:
+
+<img src=./images/pvui.png alt="Paraview UI" width=1024 />
+
+To connect to the ParaView server, choose `Connect` from the `File menu`
+
+<img src=./images/empty-connection-chooser.png alt="Empty Connection Configuration" width=512 />
+
+Now click `Add Server` and configure a server named _ofdemo_ that uses the ssh tunnel you created earlier
+
+<img src=./images/connection-config.png alt="Configure Connection" width=512 />
+
+Click `Configure` and the new server configuration should be available. Choose it and click `Connect`
+
+<img src=./images/connection-chooser.png alt="Chose Connection" width=512 />
+
+The new connection will appear in the `Pipeline Browser`
+
+<img src=./images/openradiossdemo-connected.png alt="Server Connected" width=256 />
+
+Now you are ready to select the simulation results you want to visualize. Choose `Open` from the `File` menu.
+
```

Comment:
Please include screen shots of selecting the `YarisOpenRadioss..vtk` group of files and one of the rendered car (before or after the crash)

Reply:
Woking on it